README.md

```diff
@@ -1,2 +1,76 @@
 # pyPolarionCli
+
 pyPolarionCli is a command-line tool designed for easy access to Polarion work items, e.g. for metric creation.
+
+[![License](https://img.shields.io/badge/license-bsd-3.svg)](https://choosealicense.com/licenses/bsd-3-clause/) [![Repo Status](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip) [![CI](https://github.com/NewTec-GmbH/pyPolarionCli/actions/workflows/ci.yml/badge.svg)](https://github.com/NewTec-GmbH/pyPolarionCli/actions/workflows/ci.yml)
+
+* [Installation](#installation)
+* [Overview](#overview)
+* [Usage](#usage)
+* [Commands](#commands)
+  * [Search](#search)
+* [Examples](#examples)
+* [Used Libraries](#used-libraries)
+* [Issues, Ideas And Bugs](#issues-ideas-and-bugs)
+* [License](#license)
+* [Contribution](#contribution)
+
+## Installation
+
+```cmd
+git clone https://github.com/NewTec-GmbH/pyPolarionCli.git
+cd pyPolarionCli
+pip install .
+```
+
+## Overview
+
+![overview](https://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/NewTec-GmbH/pyPolarionCli/main/design/UML/context.puml)
+
+## Usage
+
+Show help information:
+
+```cmd
+pyPolarionCli --help
+```
+
+## Commands
+
+### Search
+
+Search for Work items on the Polarion Server.
+The query must be in the Polarion format.
+
+Example:
+
+```cmd
+pyPolarionCli --user my_username --password my_password --server my_server search --project my_project --query "my_query"
```

Comment:
maybe use something like `--query "author.id:myname"` instead of `--query "my_query"`, so the usage is more obvious?